src/assets/samples/courses.json

```diff
@@ -313,5 +313,136 @@
         ]
       }
     ]
+  },
+  {
+    "id": "rem-java-21-2",
+    "coach": "Fabian Schmauder",
+    "startDate": "2021-04-19",
+    "endDate": "2021-07-13",
+    "location": "Köln",
+    "topics": [
+      {
+        "title": "Shell",
+        "items": [
+          "Unix Commands", 
+          "Git Commands", 
+          "Zsh"
+        ]
+      }, 
+      {
+        "title": "HTML5",
+        "items": [
+          "HTML5-Elemente",
+          "Formulare und Inputs"
+        ]
+      }, 
+      {
+        "title": "CSS3",
+        "items": [
+          "Selektoren",
+          "display: flex",
+          "display: grid"
+        ]
+      }, 
+      {
+        "title": "JavaScript",
+        "items": [
+          "ECMAScript 2020",
```

Comment:
Why not 2021/next?

Reply:
changed it. Was copy paste from old certificate